### ISSUE TITLE
Separate Traverse mixin on Validated instances

### ISSUE THREAD

**Issue by leandrob13** [typelevel/cats#1553]
The Traverse mixin with ApplicativeError adds a unnecessary Semigroup restriction to the Invalid side. This issue will be solved on #1548

```scala
implicit def catsDataInstancesForValidated[E](implicit E: Semigroup[E]): Traverse[Validated[E, ?]] with ApplicativeError[Validated[E, ?], E] =
    new Traverse[Validated[E, ?]] with ApplicativeError[Validated[E, ?], E]
```

**Comment by ceedubs** [comment]
#1548 was merged. Is this now resolved?

**Comment by ceedubs** [comment]
It looks to me like this was solved by #1548, so I'm going to go ahead and close it out. @leandrob13 please open this back up and let me know if I'm mistaken.
